go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/spiffe/tornjak
 
 go 1.22
 
+toolchain go1.22.2
```

Comment:
unsure if this is necessary? i'm curious was there an issue that needed this change?

Reply:
Yes I was getting and that solved it. 
```
go run cmd/manager/main.go 
go: downloading go1.22 (darwin/arm64)
go: download go1.22 for darwin/arm64: toolchain not available
```